src/main/java/com/aws/greengrass/easysetup/DeviceProvisioningHelper.java

```diff
@@ -273,17 +274,19 @@ public void cleanThing(IotClient client, ThingInfo thing, boolean deletePolicies
      *
      * To support HTTPS proxies and other custom truststore configurations, append to the file if it exists.
      */
-    private void downloadRootCAToFile(File f) {
+    private void downloadCAToFile(File f, String... urls) {
```

Comment:
nit: `downloadCAsToFile`

Reply:
Lets update the doc description for this method to reflect that this may download multiple CAs to file